### ISSUE TITLE
How does one create a Webhook?

### ISSUE THREAD

**Issue by testworksau** [ubuntu/sync-issues-github-jira#9]
Came across this useful looking Action today, but am a bit confused as to what the Webhook setup is meant to look like.

In Jira cloud, we can create an Action of trigger Type "Incoming webhook", that has the following three options and I'm not sure which one I should choose:

<img width="739" alt="image" src="https://user-images.githubusercontent.com/11973031/208362254-cada1c02-055b-4764-bc7b-f0e1542dfd3a.png">

Moving onto the next step, we need to add a component... I'm not too sure what to add here:
<img width="944" alt="image" src="https://user-images.githubusercontent.com/11973031/208362445-b449ea40-da5d-469a-8537-1882d0697967.png">

**Comment by didrocks** [comment]
Hey @testworksau. You need to select the third action (no Issues from the webhook), and then, you can get the action type and selecty what to do based on its content: create a new issue, select one and update it, delete one…

**Comment by AlexSkrypnyk** [comment]
<img width="1134" alt="Automation_-_JIRA" src="https://user-images.githubusercontent.com/378794/228399757-338533a2-9ec1-46b1-b322-6e1db98120ab.png">

**Comment by mcascone** [comment]
Is this the official process for creating the webhook to listen to this Action's events? It doesn't seem terribly helpful if we have to generate a unique automation on the Jira side.

**Comment by mai-soup** [comment]
+1, got creation working (by adding the conditional for action, thanks @testworksau for showing us that but I also had to change the value to Create, since that's apparently what the action uses even though it's not documented), but editing/moving to another status seems to require additional steps.

Not too familiar with Jira, tried setting additional fields in the issues with the GitHub URL to then find an issue with JQL to update, but keep getting invalid JSON errors for the additional fields, so I'm kinda giving up on that until further updates are posted here.

Here's the "additional fields" code I tried using (with and without quotes around the template variable):

```json
{
   "fields": {
     "GitHub URL":  {
       "value": "{{ webhookData.id }}",
    }
  },
}
```

I've now created around 15 issues just to test this action and couldn't get it to work past creating a new issue in Jira when a new one is created in GitHub. More screenshots, please.

**Comment by jszwedko** [comment]
Agreed, I'm just trying to set this action up today and I'm unsure how to configure the webhook in Jira to work correctly.

**Comment by ImTheKai** [comment]
I struggle with the same thing. The creation of a new GitHub issue creates a Jira issue if I configure the webhook correctly and set it to "Create" and use the data from the POST. How did you envision updating and deleting existing issues? Where do you store the Jira ID, that needs to be used? Looking at the current post, there is no correlation, so I wonder how this actually worked?
